bin/i18n/sync-in.rb

```diff
@@ -291,7 +291,7 @@ def localize_level_content(variable_strings, parameter_strings)
       # We want to make sure to categorize HoC scripts as HoC scripts even if
       # they have a version year, so this ordering is important
       script_i18n_directory =
-        if ScriptConstants.unit_in_category?(:hoc, script.name)
+        if Script.unit_in_category?('hoc', script.name)
```

Comment:
makes sense that this is safe to use here, because `Script` is accessed earlier 👍